src/Compilers/CSharp/Test/Semantic/Semantics/DeconstructionTests.cs

```diff
@@ -2472,10 +2472,11 @@ static void Main()
             string expectedOperationTree = @"
 IDeconstructionAssignmentOperation (OperationKind.DeconstructionAssignment, Type: System.Void, IsInvalid) (Syntax: 'var (x1, x2) = null')
   Left: 
-    ITupleOperation (OperationKind.Tuple, Type: (var x1, var x2), IsInvalid) (Syntax: 'var (x1, x2) = null')
-      Elements(2):
-          ILocalReferenceOperation: x1 (IsDeclaration: True) (OperationKind.LocalReference, Type: var, IsInvalid) (Syntax: 'x1')
-          ILocalReferenceOperation: x2 (IsDeclaration: True) (OperationKind.LocalReference, Type: var, IsInvalid) (Syntax: 'x2')
+    IDeclarationExpressionOperation (OperationKind.DeclarationExpression, Type: (var x1, var x2), IsInvalid) (Syntax: 'var (x1, x2)')
```

Comment:
Can you please file a bug to rename IDeclarationExpressionOperation to IDeclarationOperation?

Reply:
I believe we intentionally kept "expression" in the name.

Reply:
We discussed this at the design meeting, and IDeclarationOperation was making it very confusing with IVariableDeclarationOperation, it is not easy to differentiate between the two. Given that declaration expressions are very special to C# expression declarations, we could not come up with a better name then leave the Expression in the name. We can bring it up again for discussion if you are not still satisfied.